src/m365/cli/commands/cli-doctor.spec.ts

```diff
@@ -83,14 +83,14 @@ describe(commands.DOCTOR, () => {
     sinon.stub(os, 'release').returns('10.0.19043');
     sinon.stub(packageJSON, 'version').value('3.11.0');
     sinon.stub(process, 'version').value('v14.17.0');
-    sinon.stub(auth.connection, 'appId').value('31359c7f-bd7e-475c-86db-fdb8c937548e');
+    auth.connection.appId = '31359c7f-bd7e-475c-86db-fdb8c937548e';
```

Comment:
I had to change it after refactoring. I wonder if it's related to the type being undefined.

Reply:
This is why we need a direct value assignment rather than a stub:

<img width="604" alt="image" src="https://github.com/user-attachments/assets/6d85059e-c562-4f08-bdbd-5a3bc4f2735d">

Reply:
Hmm, that sucks.